artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/user-stories/components/user-story-list-item/user-story-list-item.component.ts

```diff
@@ -31,6 +31,9 @@ export class UserStoryListItemComponent {
   @Input()
   public userStory: UserStory;
```

Comment:
to samo, widze że to już jest na masterze :?

Reply:
poprawilem w moich klasach ale nie chce mi sie grzebac i refactorowac logike w innych miejscach

Reply:
Spoko